env/set_machine.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+MACHINE="$1"
+PLATFORM="${MACHINE}"
+
+printf "PLATFORM(MACHINE)=${PLATFORM}\n" >&2
```

Comment:
It's not completely obvious if we need this any more. Maybe I'm missing the point, though?

Reply:
I guess it is only used to set the `MACHINE` and `PLATFORM` env variables now so I might as well move it to the script that calls set_machine.